src/Forms/HTMLEditor/TinyMCEConfig.php

```diff
@@ -875,7 +875,7 @@ public function getTinyMCEResource()
     }
 
     /**
-     * @deprecated 4.0.0:5.0.0
+     * @deprecated 4.0.0
```

Comment:
Add the message here and update the version in the notice below.

Reply:
Updated